### PR TITLE
Add  gcl_writer connector reference doc

### PR DESCRIPTION
**Pull request by darach**
Signed-off-by: Darach Ennis <darach@gmail.com>

### CODE REVIEW

File: docs/reference/connectors/gcl_writer.md [hunk 1]
```diff
@@ -0,0 +1,239 @@
...
+## Configuration
+
+```tremor
+define connector gcl_writer from gcl_writer
+with
+    config = {
+        "connect_timeout": 1000000, # defaults to 1 second
+        "request_timeout: 1000000   # defaults to 10 seconds
+    }
+```
```
Comment (Licenser) [inline docs/reference/connectors/gcl_writer.md:22]: this is missing an `end` at the end, and it'd be nice to use `nonos::from_seconds()`

File: docs/reference/connectors/gcl_writer.md [hunk 2]
```diff
@@ -0,0 +1,239 @@
...
+  use tremor::system;
+  use integration;
+  use google::cloud::logging as gcl;
+
+  # We use a metronome as an event source in this 
+  # example. We fire periodic events every 500 
+  # milliseconds
+  define connector metronome from metronome
+  with
+    config = {"interval": 500}
```
Comment (Licenser) [inline docs/reference/connectors/gcl_writer.md:134]: interval is in nanoseconds, lets use `nanos::from_millis(500)`

File: docs/reference/connectors/gcl_writer.md [hunk 3]
```diff
@@ -0,0 +1,239 @@
...
+      # This is not a test
+      # "partial_success": false,
+      # This is not a dry run
+      # "dry_run": false,
+
+      # 500ms connection timeout
+      "connect_timeout": 500000000,
+
+      # 1s request timeout
+      "request_timeout": 1000000000,
```
[the remark this answers is not in the document]
Comment (darach) [reply docs/reference/connectors/gcl_writer.md]: 👍

File: docs/reference/connectors/gcl_writer.md (hunk 1 above)
Comment (darach) [reply docs/reference/connectors/gcl_writer.md]: Both 👍

File: docs/reference/connectors/gcl_writer.md [hunk 4]
```diff
@@ -13,11 +13,12 @@ allows json logs to be published to Cloud Logging.
...
 
 ```tremor
+use std::time::nanos;
 define connector gcl_writer from gcl_writer
 with
     config = {
-        "connect_timeout": 1000000, # defaults to 1 second
-        "request_timeout: 1000000   # defaults to 10 seconds
+        "connect_timeout": nanos::from_seconds(1), # defaults to 1 second
+        "request_timeout: nanos::from-seconds(10),# defaults to 10 seconds
```
Comment (Licenser) [inline docs/reference/connectors/gcl_writer.md:21]:
```suggestion
        "request_timeout: nanos::from_seconds(10),# defaults to 10 seconds
```

**Review by Licenser — DISMISSED** [review]

File: docs/reference/connectors/gcl_writer.md (hunk 4 above)
Comment (Licenser) [reply docs/reference/connectors/gcl_writer.md]: made my day, I found a typo :D that makes it 10000000 made vs. 1 found 🥇

**Pull request merged**